src/client.js

```diff
@@ -21,13 +22,24 @@ const viewBuildTypes = [
 const client = mozaik => {
 
     mozaik.loadApiConfig(config);
+    const caFilePath = config.get('jenkins.customCa');
+    let _certificate;
+
+    if (caFilePath.length > 0) {
+        _certificate = fs.readFileSync(caFilePath)
```

Comment:
@plouc OK, then I will catch the error and just continue instead, how about that?

```
╰─○ node app.js
info: registered API "github"
info: registered API "travis"
info: registered API "weather"
error: [jenkins] an error occurred while trying to read custom certificate (Error: ENOENT: no such file or directory, open 'does_not_exist.crt')
info: registered API "jenkins"
info: serving static contents from /Users/mathias.klippinge/src/panda-dashboard/build
info: Mozaïk server listening at http://0.0.0.0:5000
```